lib/src/RedisChannel.cpp

```diff
@@ -12,7 +12,7 @@ using namespace sairedis;
 /**
  * @brief Get response timeout in milliseconds.
  */
-#define REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS (60*1000)
+#define REDIS_ASIC_STATE_COMMAND_GETRESPONSE_TIMEOUT_MS (5*60*1000)
```

Comment:
Increasing to 5 minutes may delay the recognition of failure from syncd. Try adjusting to lower levels, if possible, after syncd initialization is complete (after create switch).

Reply:
Rita: Might  be even better if this 5min timeout is for chassis only.

Reply:
We'll undo this in this PR. In a separate PR, we'll change the time to 5 minutes only for switch_type = "voq" and the timeout will be changed back to 1 min after SAI initialization is complete.